### PR TITLE
Augi

### PR DESCRIPTION
**Pull request by adliebster**
I have added an array in the partner.component.ts file which holds the name, phone number, email, title, and affiliated organization of the point of contact the CCL has to organizations. In the associated HTML file, I have used an *ngFor to display all of the members of the above mentioned array except for the affiliated organization, which is hidden.

### CODE REVIEW

File: src/app/landing/landing.component.html [hunk 1]
```diff
@@ -6,4 +6,7 @@ <h2>New to the CCL?</h2>
   <p>
     To get permission to access the database contact <strong>steinberg@rice.edu</strong>
   </p>
+  <p>
+    Augi test.
+  </p>
```
Comment (amarshall45676) [inline src/app/landing/landing.component.html]: You can remove this.

File: src/app/partners/partner/partner.component.html [hunk 2]
```diff
@@ -17,4 +17,11 @@
       </mat-card>
     </mat-card>
   </div>
+  <div *ngFor="let contact of contacts" class="belowdiv">
+    Partner Name: {{contact.name}}
```
Comment (amarshall45676) [inline src/app/partners/partner/partner.component.html]: These should be displayed on different lines try adding a <br> in between lines.

**Review by amarshall45676 — CHANGES_REQUESTED** [review]
So this is a review of the code you just committed. Take a look at the comments I left, and make the requested changes. Other than these two it LGTM!

**Comment by adliebster** [comment]
Ok I made the requested changes and pushed the new code.

File: src/app/partners/partner/partner.component.ts [hunk 3]
```diff
@@ -33,5 +56,39 @@ export class PartnerComponent implements OnInit {
...
+  }
+
+  public createContact() {
+    if (!this.contactForm.valid){
+      console.log('Form is not valid');
+    }
+    else{
+      console.log(this.contactForm.value)
+    }
+    // this.createEntity('contact', this.contactForm).then((entityResult: Object) => {
```
Comment (amarshall45676) [inline src/app/partners/partner/partner.component.ts:80]: Try your best to keep commented out code out of the commits. I will be sure to take it out before this is merged though.

**Review by amarshall45676 — APPROVED** [review]
LGTM

**Pull request merged**